### DISCUSSION TITLE
Unpublish ORG Entries of winners in the notice PDF/HTML - Results

### DISCUSSION THREAD

**Discussion by lepo** [OP-TED/eForms-SDK#769] [Q&A] [answered]
Hello,

We are able to unpublish winners in the results (Point 6 in the notice PDF/HTML) but in the list of all Organizations the companies appears with the full data. So everyone can see the winners of the award.

How we can unpublish the winners in the list of organizations in the PDF/HTML Point 8.x

Please help, Thank you.

Best,
Günter

**Comment by YvesJo** [comment]
Hi,
There is currently no such option in place. The current "unpublish" approach is not adapted and, given the complexity of the interdependency between the various elements involved (Contract, Tender, Result, Tendring Party and Organisation), would require many actions on numerous fields to be effective. A new approach is to be expected and will most likely be implemented with SDK 2 only.

**Reply by lepo** [reply]
In case of publishing a notice for a framework agreement it's prohibited by Austrian law to publish the winners. So how to publish result notices for framework agreements?

Best,
Günter

**Reply by YvesJo** [reply]
I'll check for the various options and will get back to you most likely on Friday

**Reply by YvesJo** [reply, marked as answer]
Hi,
As there is currently no technical option available to do so, the only possibility is to have "Not to be published" specified  instead of content of type text and specific meaningless values for other content types (e.g. email, phone number ...). We will identify and share the values to be used in that last case.
The justification for not disclosing the information should also be specified in the additional information field using one of the labels of the codes of the code list associated to "Unpublished Justification Code (BT-197).

**Reply by YvesJo** [reply]
No further recommendation identified so far.
